Sources/SwiftProtobufCore/BinaryDelimited.swift

```diff
@@ -29,8 +29,8 @@ public enum BinaryDelimited {
     /// read/written.
     case truncated
 
-    /// When decoding (parsing), the length is larger than what will fit in
-    /// an Int for the compiled platform, so the message can't be parsed.
+    /// Messages are limited by the protobuf spec to 2GB, if the length says
+    /// the payload is over 2GB, this error is raised.
     case decodeTooLarge
```

Comment:
Should we rename this so that all the error constants are uniformly `tooLarge`?

Reply:
Delimited has on error type, not a split for decode and encode. So that's why I originally stuck decode in there.  Should we split it also?

Reply:
Actually, we could just use the decoding error one; it didn't exist when this was added the other week.